Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM alpine:3.17.0
-RUN apk --no-cache add ca-certificates git
+RUN apk --no-cache add ca-certificates git && \
```

Comment:
We used to tried switching non-root user, but it didn't work well.
https://github.com/aquasecurity/trivy/issues/580